pandas/tests/indexes/timedeltas/test_arithmetic.py

```diff
@@ -646,10 +645,10 @@ def test_timedelta_ops_with_missing_values(self):
         actual = df1 - timedelta_NaT
         tm.assert_frame_equal(actual, dfn)
 
-        actual = df1 + NA
-        tm.assert_frame_equal(actual, dfn)
-        actual = df1 - NA
-        tm.assert_frame_equal(actual, dfn)
+        with pytest.raises(TypeError):
```

Comment:
why is this raising? this is a big change if you don't allow nan to act as NaT in ops

Reply:
This is the current behavior for Series and Index.

Reply:
this needs a subsection in the whatsnew then, marked as an api change.